### PR TITLE
Banjo can now be worn on your back

### PR DESCRIPTION
**Pull request by z9ctn4fn**
## About the PR
You can now wear the banjo on your back.

## Why / Balance
#29616

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: You can now wear the banjo on your back and suit storage slots.
- add: Banjo sprites on your back added courtesy of my own shitty art.

### CODE REVIEW

**Comment by z9ctn4fn** [comment]
Ah shoot
Forgot about that

File: Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json [hunk 1]
```diff
@@ -18,5 +18,13 @@
         "name": "inhand-right",
         "directions": 4
       }
+      {
+        "name": "equipped-SUITSTORAGE"
+        "directions": 4
+      }
```
Comment (deathride58) [inline Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json:24]: Y'need commas
```suggestion
      },
      {
        "name": "equipped-SUITSTORAGE"
        "directions": 4
      },
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json (hunk 1 above)
Comment (z9ctn4fn) [reply Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json]: i fixed it

File: Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json [hunk 2]
```diff
@@ -17,6 +17,16 @@
       {
         "name": "inhand-right",
         "directions": 4
+      },
+      {
+        
+        "name": "equipped-SUITSTORAGE",
+        "directions": 4,
+        "comment": "lol i made these shitty sprites in like 2 minutes -archangel"
```
Comment (deathride58) [inline Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json:25]: This `comment` field is not defined within the [RSI spec](https://docs.spacestation14.com/en/specifications/robust-station-image.html); additional credits should be added to the copyright field at the root of the json object

File: Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json [hunk 3]
```diff
@@ -17,6 +17,16 @@
       {
         "name": "inhand-right",
         "directions": 4
+      },
+      {
+
```
Comment (deathride58) [inline Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json:22]: This extra line needs removal

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json [hunk 4]
```diff
@@ -17,6 +17,15 @@
       {
         "name": "inhand-right",
         "directions": 4
+      },
+      {
+        "name": "equipped-SUITSTORAGE",
+        "directions": 4,
+        "copyright": "lol i made these shitty sprites in like 2 minutes -archangel"
```
Comment (Tayrtahn) [inline Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json:24]: Not like that either. Add it to the copyright field on line 8

**Review by UbaserB — COMMENTED** [review]
The banjo sprites need to be improved, mainly in consistency and form, look at the original sprites for reference.

**Comment by slarticodefast** [comment]
Are you coming back to this?

**Comment by slarticodefast** [comment]
If you come back to this, leave a comment and I will re-open the PR.

**Comment by z9ctn4fn** [comment]
I will not be coming back to this.

**Pull request closed without merging**